gslb/test/ingestion/lib.go

```diff
@@ -206,7 +206,7 @@ func WaitAndVerify(t *testing.T, keyList []string, timeoutExpected bool, wqKeyCh
 	return true, ""
 }
 
-func addGSLBTestConfigObject(obj interface{}, f ingestion.InitializeGSLBMemberClustersFn) {
+func addGSLBTestConfigObject(obj interface{}, f ingestion.InitializeGSLBMemberClustersFn) error {
```

Comment:
Do we need the return value of error here? We don't seem to be returning an error.